api/api/openapi.bundle.yaml

```diff
@@ -172,7 +177,9 @@ paths:
         schema:
           format: int32
           type: integer
-      - in: query
+      - description: Searches ensembler name used for the ensembling job for a partial
```

Comment:
Isn't it just ensembling job name? 😄 The job name is generated from the name of the ensembler, with some suffix. But ultimately, the search is carried out on the ensembling job's name.

Reply:
Oh that's true too xD Haha alright I'll simplify the name back to what it should plainly be 😅